### ISSUE TITLE
change event does not trigger in custom headers (0.20.1+)

### ISSUE THREAD

**Issue by noresources** [handsontable/handsontable#3220]
I made a custom header with a <select> box and bind the "change" event to it.
Everything was fine in 0.20.0.
Since 0.20.1, the "change" event is never triggered and the <select> lose focus after a couple of seconds

Here is the code snippet of my custom header

```
afterGetColHeader : function(col, th) {
    th = jQuery(th);
    var menu = th.find('.view-translations-language-menu');
    if (menu.length > 0) { // already created
        return;
    }
    // Construct a select menu and its options
    menu = self.makeLanguageMenu({
        'class' : 'view-translations-language-menu form-control',
        'data-col' : (col - 1), (col - 1));
    th.append(menu);

    menu.on('change', function(e) {
        console.log('change');
        // ...
    });
}
```

**Comment by AMBudnik** [comment]
Hi @noresources can you create a demo where we can easily check this one?

**Comment by noresources** [comment]
Using 0.20.0 (working): https://jsfiddle.net/p153kbfe/
Using 0.20.1 (not working): https://jsfiddle.net/8cpmynsj/

Changing the header menu value should write "change" at the last row of the talbe

**Comment by AMBudnik** [comment]
I couldn't reproduce it with the latest 0.24.1 version I assume that it's working fine as in 0.20.0
I'm closing this issue but if I missed something important please leave a comment.

**Comment by noresources** [comment]
Sorry but I just updated my previous demo with 0.24.1 and it still doesn't work as expected 

0.20.0 (working) https://jsfiddle.net/p153kbfe/
0.20.1 (fail) https://jsfiddle.net/8cpmynsj/2/
0.24.1 (fail) https://jsfiddle.net/e7dyqLmb/2/

The expected behavior should change the last row content when the header menu value is changed.

**Comment by AMBudnik** [comment]
Thanks for an update @noresources 
I didn't tried it with FF befre and it's there.

**Comment by AMBudnik** [comment]
Hi @noresources 
can you check if it's still replicable in 0.30.1?

**Comment by noresources** [comment]
Hi @AMBudnik 
The issue has been fixed in 0.29, following issue #3555.
I updated my test with 0.30.1 (works) https://jsfiddle.net/ph3jdt3x/

**Comment by AMBudnik** [comment]
Great! Thanks for a prompt reply and the demo.
I guess we can close this issue.
